src/account/utils/Types.ts

```diff
@@ -210,21 +211,31 @@ export type NonceOptions = {
 
 export type SimulationType = "validation" | "validation_and_execution"
 
-export type Overrides = {
-  /* Value used by inner account execution */
-  callGasLimit?: Hex
-  /* Actual gas used by the validation of this UserOperation */
-  verificationGasLimit?: Hex
-  /* Gas overhead of this UserOperation */
-  preVerificationGas?: Hex
-  /* Maximum fee per gas (similar to EIP-1559 max_fee_per_gas) */
-  maxFeePerGas?: Hex
-  /* Maximum priority fee per gas (similar to EIP-1559 max_priority_fee_per_gas) */
-  maxPriorityFeePerGas?: Hex
-  /* Address of paymaster sponsoring the transaction, followed by extra data to send to the paymaster ("0x" for self-sponsored transaction) */
-  paymasterData?: Hex
-  /* Data passed into the account along with the nonce during the verification step */
-  signature?: Hex
+/**
+ * Represents offset values.
+ * @remarks
+ * An offset value determines the degree of adjustment or displacement.
+ * - `1` represents no offset (0% adjustment).
+ * - `2` represents full offset (100% adjustment).
+ * - Values between `1` and `2` indicate partial adjustments.
+ * @public
+ */
+
+/**
+ * Increment gas values by giving an offset, the given value will be an increment to the current estimated gas values, not an override.
+ * @typedef GasOffset
+ * @property {number} [callGasLimitIncrement] - Value used by inner account execution.
```

Comment:
Can you better describe this? I don't understand what value I should set here from this description